testing/CMakeLists.txt

```diff
@@ -46,6 +46,9 @@ if (CDAT_BUILD_MATPLOTLIB)
   add_subdirectory(matplotlib)
 endif()
 
+# Test ipython
+add_test(run_ipython ${CMAKE_INSTALL_PREFIX}/bin/ipython -V)
```

Comment:
@doutriaux1 you might want to tweak this "test".. I'm not sure how the return codes are dealt with, and whether this is actually an effective "test"